cloudscript/cloudscript/CreateVM.ps1

```diff
@@ -0,0 +1,137 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # VM name to be created
+    [string]$vmName,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
+    # VM resource group for above VM
+    [string]$vmResourceGroup,
+
+    [Parameter(Mandatory=$True)]
+    [ValidateNotNullOrEmpty()]
```

Comment:
Does this section involve creating an image from a VHD and creating a VM out of the subsequent image which was created? Asking this because we are asking for image specific parameters here

Reply:
No it is only creating a VM. We are asking for image name and image resource group here because thats what will be used for creating the VM. Its inline with the portal experience i.e. when user wants to create a VM, he specified which image to use for VM creation.